dsa/DSA || GAS STATION(GREEDY) #33.py

```diff
@@ -0,0 +1,20 @@
+def gascheck(j,counter,remaining_gas,cost_for_next_station):
+        if(counter==5):
```

Comment:
Why `counter == 5`? What does counter denote?

Reply:
Yes, sir, I should have used counter==n. The counter is used to determine the moment when a cycle is completed.

Reply:
Even after changes, the output always comes out -1.